gulpFile.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const gulp = require('gulp');
+const lint = require('gulp-eslint');
+const mocha = require('gulp-mocha');
```

Comment:
Some of these dependencies are not in your `package.json`. Keep in mind the differences between `dependencies` and `devDependencies` when you add them.